heron/tools/cli/src/python/response.py

```diff
@@ -33,14 +34,17 @@ class Status(object):
   Ok = 0
   InvocationError = 1
   HeronError = 100
+  DryRun = 200
 
 def status_type(status_code):
   if status_code == 0:
     return Status.Ok
   elif status_code < 100:
     return Status.InvocationError
-  else:
+  elif status_code >= 100 and status_code < 200:
```

Comment:
in my opinion, when new return code class is added (for example, dry-run here), we use number larger than the current largest number (100). I am using 200 here. 130, 150, or other numbers larger than 100 is OK.